jupyterlab_comments/src/index.ts

```diff
@@ -14,20 +14,117 @@
  * limitations under the License.
  */
 
+
 import {
   JupyterFrontEnd,
   JupyterFrontEndPlugin,
+  ILabShell,
 } from '@jupyterlab/application';
 
+import {
+    IDocumentManager,
+} from '@jupyterlab/docmanager';
+
+import {
+  Widget
+} from '@phosphor/widgets';
+
+import {
+  Message
+} from '@phosphor/messaging';
+
+import { MainAreaWidget, ICommandPalette } from '@jupyterlab/apputils';
+
+import { PageConfig } from '@jupyterlab/coreutils';
+
+import { httpGitRequest } from './git'
+
+class File {
```

Comment:
I plan to refactor the File and CommentsWidget classes to mimic the new cookie cutter, in another commit